src/screens/SearchResults.js

```diff
@@ -0,0 +1,96 @@
+import React, { Component } from "react";
+import map from "../assets/images/map.PNG";
+import { ListGroup } from "react-bootstrap";
+import CategoryButtonGroup from "../components/CategoryButtonGroup/CategoryButtonGroup";
+import EventLink from "../components/EventLink";
+import { clearAllBodyScrollLocks } from 'body-scroll-lock';
+import Map from "../components/maps/Map";
+import { withStore } from "../assets/helpers/store";
+import {withRouter, NavLink} from 'react-router-dom'
+
+class SearchResults extends Component {
+   constructor(props) {
+      super(props);
+      this.state = {
+         viewOnly: true,
+         selectedCategory: '',
+         events: [],
+         loggedInUserName: 'User',
+         userLoggedIn: 'false',
+         loggedInUser: {},
+         tggle: false
+      }
+   }
+
+   componentWillMount() {
+      
+               this.setState({
+                  events: this.props.results
+                  
+               })
+            
+   }
+
+   componentDidUpdate(prevProps, prevState) {
+      if ((this.loggedInUser !== this.props.store.loggedInUser) && (this.state.tggle == false)) {
+         this.setState({ loggedInUser: this.props.store.loggedInUser })
+         this.setState({ tggle: true })
+         if (this.state.loggedInUserName === 'User') {
+            this.state.loggedInUserName = this.props.store.loggedInUser.userFirstName
+         }
+      }
+      if(this.props.results!==this.state.events){
+         this.setState({events:this.props.results})
+      }
+   }
+
+
+   render() {
+      let eventElements = []
+      if (this.props.results === []) {
+         eventElements.push(
+            <div className="noEventContatiner">
+               <p className="noEventMsg">No Events Found</p>
+            </div>
+         )
+      } else {
+
+         this.props.results.forEach(function (event, index) {
+            eventElements.push(
+               <EventLink
+                  key={index}
+                  name={event.name}
+                  category={event.category}
+                  host={event.creator}
+                  date={event.date}
+                  startTime={event.startTime}
+                  endTime={event.endTime}
+                  location={event.location}
+                  description={event.description}
+                  spotsAvailable={event.availableSpots}
+                  attendees={event.attendees}
+                  coordinates={event.coordinates}
+               />
+            )
+         })
+      }
+
+      return (
+         <div className="homePageBanner" >
```

Comment:
We should display our please sign in modal if they are searching without being logged in